go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
 	github.com/vmware-tanzu/carvel-kapp-controller v0.27.0
-	golang.org/x/net v0.0.0-20210520170846-37e1c6afe023
+	golang.org/x/net v0.0.0-20220906165146-f3363e06e74c
```

Comment:
Should we bump it to the latest version?

Reply:
This is the version in which trivy scan mentioned vulnerability is fixed.

Reply:
> This is the version in which trivy scan mentioned vulnerability is fixed.

It would remain fixed in the newer versions also, right?